### ISSUE TITLE
pam_module!() macro requires importing PamFlags type

### ISSUE THREAD

**Issue by Foxcirc** [rcatolino/pam_sm_rust#32]
# Example

```rust
use pamsm::{PamServiceModule};
struct Pam;
impl PamServiceModule for Pam {}
pamsm::pam_module!(Pam); // ERROR: PamFlags used in the macro but not imported
```
This is not a huge bug but is also probably really easy to fix.
It seems PamFlags is used in `pam.rs:239`.

Thanks for writing this great crate!

**Comment by rcatolino** [comment]
Hello, I've added the pamsm qualifier to the PamFlags use in the macro, you shouldn't need to import it manually anymore. Can you confirm it works on master ?

**Comment by Foxcirc** [comment]
Yes, it works for me now.
I also noticed you have to import the PamServiceModule trait manually (similar to the other one), otherwise there is a pretty confusing error message about missing functions that originates in the pam_module!() macro as well.

**Comment by rcatolino** [comment]
Hum this one is more tricky, the `PamServiceModule` needs to be in scope in the macro, I could add a `use pamsm::PamServiceModule` in the macro block, but then any code calling the `pam_module!` macro that *already* has imported that trait will fail because `PamServiceModule` will be imported multiple times.

I'm not familiar enough with macros to find a good solution for that, and the error message literally tells you that you need to `use pamsm::PamServiceModule;` so it's probably not a big deal.

I've published version 0.5.4 with that fix.
